modules/light-clients/08-wasm/types/client_state.go

```diff
@@ -261,5 +261,8 @@ func call[T ContractResult](ctx sdk.Context, clientStore sdk.KVStore, cs *Client
 	if !output.Validate() {
 		return output, errorsmod.Wrapf(errors.New(output.Error()), "error occurred while calling contract with code ID %s", hex.EncodeToString(cs.CodeId))
 	}
+	if len(out.Messages) > 0 {
+		return output, errorsmod.Wrapf(ErrWasmSubMessagesNotAllowed, "code hash (%s)", hex.EncodeToString(cs.CodeId))
```

Comment:
I mention code hash in the message, assuming that [this PR](https://github.com/cosmos/ibc-go/pull/4091) will eventually get merged.

Reply:
The var naming of `out` doesn't really feel intuitive to me, but we can rename it to whatever later, also maybe we could add something like this in a future improvement - `handleContractResponse()` [ref](https://github.com/CosmWasm/wasmd/blob/main/x/wasm/keeper/keeper.go#L983)